sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationIdentifier.java

```diff
@@ -23,27 +98,40 @@ public static CommunicationIdentifier fromRawId(String rawId) {
             throw new IllegalArgumentException("The parameter [rawId] cannot be null to empty.");
         }
 
-        if (rawId.startsWith("4:")) {
+        if (rawId.startsWith(PHONE_NUMBER)) {
             return new PhoneNumberIdentifier(rawId.substring("4:".length()));
```

Comment:
Use the constant PHONE_NUMBER here as well.
`return new PhoneNumberIdentifier(rawId.substring(PHONE_NUMBER.length()));`